addons/web/static/src/js/core/ajax.js

```diff
@@ -322,12 +322,19 @@ function get_file(options) {
     return true;
 }
 
-function post (controller_url, data) {
+function post (controller_url, data, { files } = {}) {
```

Comment:
pass form to FormData constructor and create the form on the xml on the other PR